jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.sh

```diff
@@ -18,4 +18,4 @@
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 
-java -jar ${SCRIPTPATH}/../lib/${project.build.finalName}.jar "$@"
+java -jar "${SCRIPTPATH}"/"${project.build.finalName}".jar "$@"
```

Comment:
What's the reason for this change?

Reply:
The cli jar does not get built into the lib directory.

Reply:
Even after I removed the pushd and popd and executed the script by ./target/jsonschema2pojo.sh it still works.

Reply:
This change will break our releases, we need to revert it.

Reply:
Fixed.